pkgs/os-specific/linux/rtl8814au/default.nix

```diff
@@ -31,9 +31,8 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "Realtek 8814AU USB WiFi driver";
-    homepage = "https://github.com/zebulon2/rtl8814au";
-    license = licenses.gpl2;
+    homepage = "https://github.com/morrownr/8814au";
+    license = licenses.gpl2Only;
     maintainers = [ maintainers.lassulus ];
-    platforms = [ "x86_64-linux" "i686-linux" ];
```

Comment:
Why is this removed?

Reply:
The fork supports `aarch64`, so no need to isolate platforms to `x86_64` and `i686`.

Reply:
well since this can't be compiled without the kernel anyway, not setting it should be fine?